### DISCUSSION TITLE
Erreur 500 quand on modifie les checkbox badges ou camera dans les Batiments/Salles

### DISCUSSION THREAD

**Discussion by WID-226** [dbarzin/mercator#730] [Ideas]
Bonjour, 

Nous avons recemment installé MERCATOR afin de cartographier efficacement notre SI.

En parcourant nous avont remarqué la creation de batiment fonctionnait sans soucis mais la mise a jour de cette derniere pour les champs "Badge et Camera" posait probleme.

Noux avons donc rajouté un champ valeur="1" aux checkbox pour corriger l'erreur.
Il renvoyait une valeur "ON" plutot que "1" ce que la BDD refusait d'enregistrer vu que le champs est en intéger.

Cordialement

**Comment by dbarzin** [comment]
Ce problème a été corrigé il y a quelques jours : https://github.com/dbarzin/mercator/issues/729
Pouvez-vous vérifier que vous utilisez la dernière version ?
Merci,
Didier
